frontend/packages/dev-console/src/plugin.tsx

```diff
@@ -283,6 +283,18 @@ const plugin: Plugin<ConsumedExtensions> = [
         )).default,
     },
   },
+  {
+    type: 'Page/Route',
+    properties: {
+      perspective: 'dev',
+      exact: true,
+      path: ['/topology-list/ns/:ns'],
```

Comment:
I'd rather see the topology url with an additional segment of `/list` at the end rather than a brand new url from scratch.